background.js

```diff
@@ -36,3 +36,29 @@ chrome.windows.onRemoved.addListener(function(windowId) {
     currentWindow = null
   }
 })
+
+chrome.browserAction.onClicked.addListener(buttonClicked)
+function buttonClicked() {
```

Comment:
I like it that you also add the browser action to click to enter the settings page! Great job!